The `skipLibCheck` flag is to fix an error during testing apparently caused by a previous update to `jsdoc`:
`Error: node_modules/@types/mdurl/build/index.cjs.d.ts(47,17): error TS2304: Cannot find name 'Omit'.`

The error is entirely unrelated to both this update, and to what's being tested, so the flag seems appropriate to fix the problem.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -13,7 +13,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node: [ 14, 16, 18]
+        node: [ 16, 18, 20]
     steps:
       - name: Checkout
         uses: actions/checkout@v3

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
   "scripts": {
     "test": "npm run test:javascript && npm run test:typescript",
     "test:javascript": "jasmine spec/**/**.spec.js",
-    "test:typescript": "tsc examples/typescript/example.ts --noEmit --strict --skipLibCheck",
+    "test:typescript": "tsc examples/typescript/example.ts --noEmit --strict",
     "jshint": "jshint lib/rest/** lib/base/** lib/http/**",
     "jscs": "eslint lib/base/**/**.js lib/http/**/**.js --fix",
     "check": "npm run jshint && npm run jscs",
@@ -81,5 +81,8 @@
   "types": "./compatibility-api.d.ts",
   "engines": {
     "node": ">=12"
+  },
+  "overrides": {
+    "@types/mdurl": "1.0.5"
   }
 }

--- a/package.json
+++ b/package.json
@@ -80,7 +80,7 @@
   },
   "types": "./compatibility-api.d.ts",
   "engines": {
-    "node": ">=12"
+    "node": ">=16"
   },
   "overrides": {
     "@types/mdurl": "1.0.5"

